src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/TrackingConsentValue.cs

```diff
@@ -0,0 +1,77 @@
+using Fluid;
+using Fluid.Values;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using OrchardCore.Liquid;
+
+#nullable enable
```

Comment:
I don't think this is needed

Reply:
I'm using some nullable annotations in this class.